newdle/models.py

```diff
@@ -1,11 +1,13 @@
 import random
+from datetime import datetime
 from enum import auto
 
 from flask import current_app
 from sqlalchemy.dialects.postgresql import ARRAY, JSONB
 from sqlalchemy.event import listens_for
 from sqlalchemy.ext.hybrid import hybrid_property
 from sqlalchemy.schema import CheckConstraint
+from sqlalchemy.sql import text
```

Comment:
nothing wrong with importing it directly from SA, but `db.text()` would work as well (flask-sqlalchemy exposes various commonly used sqlalchemy symbols via `db`)